content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -220,7 +220,7 @@ are true. The following taints are built in:
     as unusable. After a controller from the cloud-controller-manager initializes
     this node, the kubelet removes this taint.
 
-In case a node is to be evicted, the node controller or the kubelet adds relevant taints
+In case a node is to be drained, the node controller or the kubelet adds relevant taints
```

Comment:
I don't think drained is the right word also as a node won't be trained when taints like disk-pressure are added